flow-typed/submissions.js

```diff
@@ -178,3 +178,14 @@ declare type GeneratePaymentConfigPayload = {
   crn2?: string,
   crn3?: string,
 }
+
+declare type NewFormSubmissionFileAccessToken = {
+  expiry: string,
+  s3Location: string,
+  submisionId: string,
+  emailAddress: string,
```

Comment:
Whats the purpose of storing this?

Reply:
We agreed just for record keeping purposes